pages/dashboard.tsx

```diff
@@ -0,0 +1,335 @@
+                      key={`trade-`}
+                      className={` border-t border-b text-left bg-white `}
+                    >
+                      <td className="px-6 py-4 whitespace-nowrap text-sm  ">
+                        {new Date(trade.createdAt).toLocaleDateString()}
+                      </td>
+                      <td className="px-6 py-4 whitespace-nowrap text-sm  ">
+                        {trade.transactionId}
+                      </td>
+                      <td
+                        className={` px-6 py-4 whitespace-nowrap text-sm  uppercase ${
+                          trade.role === Role.buyer
+                            ? "text-green-500"
+                            : "text-red-500"
+                        } `}
+                      >
+                        {trade.role}
+                      </td>
+                      <td className="px-6 py-4 whitespace-nowrap text-sm  text-link ">
+                        {trade.user}
+                      </td>
+                      <td className="px-6 py-4 whitespace-nowrap text-sm  flex items-center gap-1">
+                        <span className="rounded-full bg-red-500 w-2 h-2" />{" "}
+                        {`$${trade.rate / 10000}`}
+                      </td>
+                      <td className="px-6 py-4 whitespace-nowrap text-sm  ">
+                        {trade.amount.toLocaleString()}
+                      </td>
+                      <td className="px-6 py-4 whitespace-nowrap text-sm  uppercase ">
+                        {trade.paymentIn}
+                      </td>
+                    </tr>
+                  ))}
+                </tbody>
+              </table>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="border-t px-6 py-4 flex justify-between items-center text-sm">
+        <span>
+          Showing <b>{trades.length / 10 >= page ? 10 : trades.length % 10}</b>{" "}
+          of <b>{trades.length}</b>
+        </span>
+        <div className="flex justify-center items-center gap-2">
+          <Button type="text" disabled={page === 1}>
+            Previous
+          </Button>
+          {(() => {
+            const buttons: ReactNode[] = [];
+            for (
+              let n = page ?? page - 1;
+              n <= (Math.ceil(trades.length / 10) < page ? page + 1 : page);
+              n++
+            ) {
+              buttons.push(
+                <button
+                  className={`${
+                    page === n ? "border border-link text-link" : ""
+                  } hover:border-2 border-link h-6 w-6`}
+                >
+                  {n}
+                </button>
+              );
+            }
+            return buttons;
+          })()}
+          <Button type="text" disabled={page >= trades.length / 10}>
+            Next
+          </Button>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export default function dashboard() {
```

Comment:
refer to FetchTrades on https://github.com/tnbCrow/Frontend/blob/main/pages/index.tsx#L190

Reply:
I changed it to reflect the api 
```ts
export interface Trade {
  created_at: string
  uuid: string
  buyer: string
  seller: string
  agent: string
  rate: number
  amount: number
  payment: PaymentOptions
}
```